Cactooz/reShuffle
# Add Album Shuffle

### Tasks complete
- Add track and disc number to track fetch
- Break out Fisher-Yates and spread function
- Add album shuffle that groups tracks in the same album and sorts them according to disc and track number. The order of albums is determined by spreading the artists
- Add timeouts and disabled states to all play/pause and next/previous buttons. 
- If the end of the queue is reached, a new shuffle is performed on the same playlist

### Testing
1. Change albumShuffle to nr 3 at the top in shuffles.js
2. Press play on a playlist using shuffle nr 3 and pay attention to the song queue
3. Test to spam click on buttons and see if there are any errors in the console
4. Go through the entire queue and watch it reshuffle
